worldlimiter.lua

```diff
@@ -1,29 +1,50 @@
+local WorldLimiter_ScheduleInProgress = 0
```

Comment:
My browser cannot render the "eyes" of your textual emoji. A screenshot or a description would be appreciated.